tests/qa_automation/kernel_multipath.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use qa_test_multipath to test multipath over iscsi
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base "qa_run";
+use strict;
+use testapi;
+use utils;
+use iscsi;
+
+sub start_testrun {
+    my $self = shift;
+
+    zypper_call("in open-iscsi qa_test_multipath");
+
+    systemctl 'start iscsid';
+    systemctl 'start multipathd';
+
+    # Set default variables for iscsi iqn and target
+    my $iqn    = get_var "ISCSI_IQN",    "iqn.2016-02.de.openqa";
+    my $target = get_var "ISCSI_TARGET", "10.0.2.1";
+
+    # Connect to iscsi server and obtain wwid for multipath configuration
+    iscsi_discovery $target;
+    iscsi_login $iqn, $target;
+    # virtio serial is too fast, system need delay to properly detect multiple paths
+    # TODO: FIX? Probably loop with check?
```

Comment:
Yes, I recommend a for-loop with shorter sleep time in between. We have some test modules that can serve as example using a similar approach, e.g. as in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/yast2_dns_server.pm#L60